ipinfo/core.go

```diff
@@ -107,6 +108,12 @@ func GetIPInfo(ip net.IP) (*Core, error) {
 // GetIPInfo returns the details for the specified IP.
 func (c *Client) GetIPInfo(ip net.IP) (*Core, error) {
 	relURL := ""
+	if ip != nil && isBogon(netip.MustParseAddr(ip.String())) {
```

Comment:
See https://pkg.go.dev/net/netip#AddrFromSlice - we wont have to go to string and then parsed.

Reply:
Even easier, if we dont wanna use this netip package in many other places, is to just accept `net.IP` but parse it to this inside the function itself.

Reply:
We can't use the `AddrFromSlice` if we use that function it converts IPv4 to IPv6 and returns `::ffff:8.8.8.8` on `8.8.8.8` input. This causes all the `IPv4` IPs to be flagged as bogon.

Reply:
Ouch, ok.